test/remote/device_tests.js

```diff
@@ -369,99 +369,6 @@ describe('remote device', function() {
-      var deviceInfo = {
-        name: 'test device',
-        type: 'mobile'
-      }
-      return Client.create(config.publicUrl, email, password, {
-        createdAt: '0'
-      })
-      .then(
-        function (client) {
-          return client.updateDevice(deviceInfo)
-            .then(
-              function () {
-                return client.devices()
-              }
-            )
-            .then(
-              function (devices) {
-                assert.equal(devices.length, 1, 'devices returned one item')
-                assert.strictEqual(devices[0].lastAccessTime, 0, 'devices returned correct lastAccessTime')
-                assert.strictEqual(devices[0].lastAccessTimeFormatted, '',
-                  'devices returned empty lastAccessTimeFormatted because lastAccesstime is 0')
-                assert.equal(devices[0].name, deviceInfo.name, 'devices returned correct name')
-                assert.equal(devices[0].type, deviceInfo.type, 'devices returned correct type')
-                return client.destroyDevice(devices[0].id)
-              }
-            )
-        }
-      )
-    }
-  )
-
-  it(
-    'devices list, sessionToken.lastAccessTime === -1',
-    () => {
-      var email = server.uniqueEmail()
-      var password = 'test password'
-      return Client.create(config.publicUrl, email, password, {
-        createdAt: '-1'
-      })
-      .then(
-        () => assert(false),
-        err => {
-          assert.equal(err.code, 400)
-        }
-      )
-
-    }
-  )
-
-  it(
-    'devices list, sessionToken.lastAccessTime === THE FUTURE',
-    () => {
-      var email = server.uniqueEmail()
-      var password = 'test password'
-      var deviceInfo = {
-        name: 'test device',
-        type: 'mobile'
-      }
-      var theFuture = Date.now() + 10000
-      return Client.create(config.publicUrl, email, password, {
-        createdAt: '' + theFuture
-      })
-      .then(
-        function (client) {
-          return client.updateDevice(deviceInfo)
-            .then(
-              function () {
-                return client.devices()
-              }
-            )
-            .then(
-              function (devices) {
-                assert.equal(devices.length, 1, 'devices returned one item')
-                assert.ok(devices[0].lastAccessTime > 0, 'devices returned correct lastAccessTime')
-                assert.ok(devices[0].lastAccessTime < theFuture, 'devices returned correct lastAccessTime')
-                assert.strictEqual(devices[0].lastAccessTimeFormatted, 'a few seconds ago',
-                  'devices returned correct lastAccessTimeFormatted')
-                assert.equal(devices[0].name, deviceInfo.name, 'devices returned correct name')
-                assert.equal(devices[0].type, deviceInfo.type, 'devices returned correct type')
-                return client.destroyDevice(devices[0].id)
-              }
-            )
-        }
-      )
-    }
-  )
-
```

Comment:
I can definitely adapt these tests to not pass _createdAt, but I don't fully understand what they are testing. Is it the behavior of updateDevice? The devices get route? I'll be sure to update the test description too once we know XD